packages/ffe-form/less/dropdown.less

```diff
@@ -35,19 +35,7 @@
     &:active {
         border-color: var(--ffe-v-input-bg-color);
         box-shadow: 0 0 0 2px var(--ffe-g-primary-color);
```

Comment:
```suggestion
        box-shadow: 0 0 0 2px var(--ffe-g-primary-color);
        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='24' viewBox='0 -960 960 960' width='24'%3E%3Cpath fill='%23002776' d='M480-373.539q-7.231 0-13.461-2.308-6.231-2.308-11.846-7.923L274.924-563.539q-8.308-8.307-8.5-20.884-.193-12.577 8.5-21.269 8.692-8.692 21.076-8.692t21.076 8.692L480-442.768l162.924-162.924q8.307-8.307 20.884-8.5 12.576-.192 21.268 8.5 8.693 8.692 8.693 21.077 0 12.384-8.693 21.076L505.307-383.77q-5.615 5.615-11.846 7.923-6.23 2.308-13.461 2.308Z'/%3E%3C/svg%3E%0A");
```

Reply:
Skjønner, takk for fiks! Er det en grunn til at det er annen farge på denne komponenten, men ikke på SearchableDropdown?

Reply:
Godt spørsmål. Og godt sett igjen! Har sjekket dette ordentlig ut i Figma nå, og det ser ut til at jeg tok feil. De skal ikke endre farge likevel, så det er SearchableDropdown som er fasiten.

Den opprinnelige fiksen din var med andre ord rett og mine forslag innførte en ny feil. Beklager det! 😅